src/servers/sapphire_zone/Script/NativeScriptApi.h

```diff
@@ -88,7 +88,7 @@ class EventScript : public ScriptObject
       ScriptObject( questId, typeid( EventScript ).hash_code() )
    { }
 
-   virtual void onTalk( uint32_t eventId, Entity::Player& player, uint64_t actorId ) { }
+   virtual void onTalk( uint32_t eventId, Entity::Player& player, uint32_t actorId ) { }
```

Comment:
the client sends us a 64bit uint with the actor id, you shouldn't be truncating it just because it doesn't make a difference at the moment

Reply:
At the moment, VS consider it like a warning and I didn't found problems ingame (atm)

Reply:
auto actor = static_cast< uint32_t >( actorId );